### DISCUSSION TITLE
Dropdown with clickable title

### DISCUSSION THREAD

**Discussion by smartm0use** [fomantic/Fomantic-UI#1947] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
Hello,
I need a dropdown menu in which user can click on the title and being redirect to the first item of the dropdown (so the first item should be highlined).

Here is a playground:
https://jsfiddle.net/orms7ad8

As you can see I have set `on` setting to `hover` so users could click on dropdown title or, after going hover on it, select one of its items. But, how to make users to click on the title?

**Comment by lubber-de** [comment]
Add `selected` to the desired items class, then it will be highlighted when opening the dropdown
See https://jsfiddle.net/lubber/o6au3c2z/2/

**Comment by smartm0use** [comment, marked as answer]
Thank you for your reply. Maybe I was not clear, anyway I reach what I need adding `selected` to dropdown item and `active` to child item.

https://jsfiddle.net/ukce5pq8
